docs/recipes/purescript.md

```diff
@@ -122,6 +122,9 @@ function maybe<A, B>(whenNone: () => B, whenSome: (a: A) => B, fa: Option<A>): B
 }
 ```
 
+Here TypeScript also provides exhaustiveness check because otherwise function
+would lacks ending return statement and return type does not include 'undefined'.
```

Comment:
```suggestion
would lack ending return statement and return type does not include 'undefined'.
```

Reply:
Oh my, sorry, will fix. 🤦

While are we doing this, wold it make sense to mention that in case of having switch otherwise, exhaustiveness check can be done with unification with never with something like

```
function assertUnreachable(x: never): never {
    throw new Error("Didn't expect to get here");
}
```
?

Reply:
This is https://github.com/gcanti/fp-ts/blob/083e8608b175804c6d5783c89a5002143b654256/src/function.ts#L274-L276

Reply:
Thank you, change included